pkg/gatherers/clusterconfig/openshift_storage.go

```diff
@@ -0,0 +1,53 @@
+package clusterconfig
```

Comment:
You can `//nolint: dupl` as the first line to fix the linting problem, but I guess you will need to add it in the `openshift_logging.go` as well.

Reply:
should be fixed by https://github.com/openshift/insights-operator/pull/614/commits/0e6c12b3c6de3c125dc1198bd897092c3570acb7